File: Reverse.c
#include <stdio.h>


int reverse(int num) {
  int rev = 0;
  
  while (num!=0) {
    int rem = num % 10;
    rev = (rev*10) + rem;
    num = num / 10;
  }
  return rev;
}
int main() {
  int num;
  printf("Enter number:");
  scanf("%d", &num);

  int result = reverse(num);
  printf("Reverse of %d is %d xyz", num, result);

}

File: Reverse.py
n = int(input('Enter a number: '))
original_n = abs(n)  # Ensure we work with the absolute value

rev = 0
while original_n != 0:
    rem = original_n % 10
    rev = rev * 10 + rem
    original_n //= 10

# If the input was negative, make the reversed number negative too
if n < 0:
    rev *= -1

print("Reversed number:", rev)
